docs/intro.rst

```diff
@@ -35,8 +35,13 @@ All dependencies can be installed using the standard package manager for Python,
 
 Compatibility
 -------------
+<<<<<<< HEAD
```

Comment:
merge conflict made its way into the commit :(

Reply:
Oops. Fixed now :)